components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/resources/admin-api.yaml

```diff
@@ -3779,6 +3830,27 @@ definitions:
         type: integer
         example: -1234
 
+  #-----------------------------------------------------
+  # The CustomURL info response resource
+  #-----------------------------------------------------
+  CustomUrlInfo:
+    title: Custom url information
+    type: object
+    description: The custom url information of the tenant domain
+    properties:
+      tenantDomain:
+        type: string
+        example: 'carbon.super'
+      tenantAdminUsername:
+        type: string
+        example: 'john@foo.com'
```

Comment:
Are these required?

Reply:
The response schema is expected to have these keys.